packages/mdc-base/component.js

```diff
@@ -77,6 +77,7 @@ export default class MDCComponent {
   // with the given data.
   emit(evtType, evtData) {
     let evt;
+    /* istanbul ignore else: Will never fire in Chrome */
```

Comment:
Meh...if it doesn't fire in certain browsers, we should hack it in our tests so it will, e.g.

```js
test('handles situation where custom events aren\'t available', () => {
  const {CustomEvent} = window;
  window.CustomEvent = undefined;
  // test code...
  window.CustomEvent = CustomEvent;
});
```